miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/util/MiniAppDatabaseEncryptionUtil.kt

```diff
@@ -155,7 +155,7 @@ internal object MiniAppDatabaseEncryptionUtil {
 
         val holder = getPasscodeHolder(context)
 
-        if (holder == null || (holder.salt == null || holder.iv == null || holder.encryptedPasscode == null)) {
+        if (holder == null) {
```

Comment:
u sure about this?

Reply:
Yeah, basically we should check only `holder == null` and we can remove the other conditions as the properties are non-nullable e.g. [EncryptedPasscodeDataHolder](https://github.com/rakutentech/android-miniapp/blob/master/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/util/MiniAppDatabaseEncryptionUtil.kt#L35~L38). Removing those redundant conditions would improve branch coverage in this part. 
However, we need to add condition in future accordingly if we change any property.